### ISSUE TITLE
[Feature Request]: Youtube Music does not send "media changed" event to NotificationCenter on MacOS

### ISSUE THREAD

**Issue by SpyMachine** [th-ch/youtube-music#2269]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Sorry I'm not very familiar with MacOS development but I'm trying to use Sketchybar to show the currently playing song with the "media_changed" event subscribed.

The event works with every other player, including using Youtube music in the browser but not for this desktop player. It looks like this is where sketchybar subscribes to the notification https://github.com/FelixKratz/SketchyBar/blob/master/src/media.m#L33-L36.

I wasn't really sure if this was a bug or an FR but figured I'd keep it at FR as it's not really a stated supported feature.

### Proposed Solution

I'm honestly not sure what the solution is or why this doesn't work as it does seem like "Now Playing" knows the song changed.

I might try to do some more research later and I'll come back with my findings or a PR if I figure out what's going on.

### Alternatives Considered

N/A

**Comment by jasoncyu** [comment]
Yeah, I experienced the same thing with sketchybar. Kind of a curious issue, because aside from this, the app behaves like any other media player

**Comment by Lalit64** [comment]
@th-ch This seems to be a common issue, but no one is replying or even trying to resolve this?

**Comment by Lalit64** [comment]
@SpyMachine I can get control center Album Cover Title and Artist working by enabling the Lumia Stream Plugin but it still somehow doesn't work with SketchyBar

**Comment by JellyBrick** [comment]
> [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/th-ch?size=32">@th-ch](https://github.com/th-ch) This seems to be a common issue, but no one is replying or even trying to resolve this?

> but no one is replying or even trying to resolve this?

Because this issue was caused by Electron's internal implementation. check the labels

**Comment by Lalit64** [comment]
@JellyBrick I can't find any issue in the https://github.com/electron/electron/ repository could you maybe link to it?

**Comment by Lalit64** [comment]
@JellyBrick Update: it seems to be working without Lumia Stream enabled, but it still doesn't show up in sketchybar. It seems that `env.INFO.app` is undefined when trying to use in sketchybar.
